cypress/downloads/response1.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Looks like this response file will be created once you've ran the code. You can remove them from the commit

Reply:
REsolved